src/util/vscodePrompts.ts

```diff
@@ -155,8 +159,24 @@ export async function askForFolder(customOptions: OpenDialogOptions): Promise<Ur
   const result = await window.showOpenDialog(Object.assign(options, customOptions));
 
   if (result && result.length > 0) {
-    return result[0];
+    const targetFolder = result[0];
+
+    if (fs.existsSync(`${targetFolder.path}/${artifactId}`)) {
```

Comment:
Use the async exists function exported from util.ts instead of the sync version
Ie. `if (await util.exists(...))`